packs/nest/actions/humidity.py

```diff
@@ -0,0 +1,14 @@
+from lib import actions
+
+class HumidityAction(actions.BaseAction):
+    def run(self, structure, device, target=None):
+        if target:
+            target_humidity = target
+        else:
+            target_humidity = self._get_device(structure, device).humidity
```

Comment:
So, I actually debated a pattern here in my head, so I'm glad you picked up on this. It does seem like a bit of an overload. But, there were two options:

* create setters/getters (`set_humidity`, `get_humidity`) and the same with temperature
* Have a single action that is capable of introspection as well as setting.

To date, I've been taking the first option, as it seems to read cleaner in a workflow if I were to describe it in human language.

I don't know what is going to make best sense really in terms of both usability now and in the future. Specifically how I might take logic decisions in a WF based on results. Certainly open to thoughts here.